blueprints/component-test/index.js

```diff
@@ -22,13 +39,43 @@ module.exports = {
   locals: function(options) {
     var dasherizedModuleName = stringUtil.dasherize(options.entity.name);
     var componentPathName = dasherizedModuleName;
-    if(options.pod && options.path !== 'components' && options.path !== '') {
+    var testImports = EOL + "import hbs from 'htmlbars-inline-precompile';" + EOL;
+    var testOptions = "integration: true";
+    var friendlyTestDescription = testInfo.description(options.entity.name, "Integration", "Component");
+    var testContent = "assert.expect(1);" + EOL + EOL +
+      "  // Set any properties with this.set('myProperty', 'value');" + EOL +
+      "  // Handle any actions with this.on('myAction', function(val) { ... });" + EOL + EOL +
+      "  this.render(hbs`{{" + dasherizedModuleName + "}}`);" + EOL + EOL +
```

Comment:
I wonder if this should be changed to show how components with a template block can be tested now? Maybe something like this:

``` js
"this.render(hbs`" + EOL
"  {{#" + dasherizedModuleName + "}}" + EOL
"    template block text" + EOL
"  {{/" + dasherizedModuleName + "}}" + EOL
"`);" + EOL + EOL +
"assert.equals(this.$().text().trim(), 'template block text');"
```

What do you think?